lib/plug/csrf_protection.ex

```diff
@@ -101,19 +101,39 @@ defmodule Plug.CSRFProtection do
 
   @behaviour Plug
 
-  def init(opts), do: opts
+  
+  def init(opts) do
+    verity_opts opts 
+  end
 
-  def call(conn, _opts) do
+  def call(conn, opts) do
+    opts = verity_opts opts
```

Comment:
`opts` argument in this call is the `opts` which come from `init/1` function and thus are already  verified.